services/scheduler/scheduler.go

```diff
@@ -0,0 +1,218 @@
+package scheduler
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/percona/pmm-managed/models"
+
+	"github.com/go-co-op/gocron"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/reform.v1"
+)
+
+type jobsDeps struct {
+}
```

Comment:
That's where we will keep dependencies on the other services?

Reply:
Yeah, I didn't came up with better solution. So eg. pass agentsService to scheduler and then to this jobs.

Reply:
Not sure, but maybe it will be more explicit if job/task will accept required deps in constructor

```
func NewEchoJob(message string, someSrv service.Service, etc)
```

And when we retore this task from DB schedule service will invoke this constructor again to rebuild task.

Reply:
I can do this. But still, I'll need to pass every service to scheduler.